src/pages/docs/creators/python/tutorials/building-custom-image.md

```diff
@@ -0,0 +1,174 @@
+
+{% alert level="info" %}
+
+This the tutorial is designed for: OS X 10.14+, Ubuntu 18.04 or 20.04, and Windows
+
+Prerequisites:
+
+- Have Docker installed and Docker service available. If you don't have Docker installed follow these [instructions](https://www.docker.com/products/docker-desktop)
+- Gvmkit-build installed ([see instructions](/docs/creators/javascript/examples/tools/gvmkit-build-installation))
+- Yagna service installed and running with `try_golem` app-key configured ([see instructions](/docs/creators/javascript/examples/tools/yagna-installation-for-requestors))
+
+{% /alert %}
+
+## Creating the Dockerfile
+
+This is the simple `Dockerfile` we are going to use, using the `Node` base image, creating one volume, and setting the working directory. Simply create a file with the name `Dockerfile`, without any file extension, and paste the following contents:
+
+```dockerfile
+FROM node:latest
+WORKDIR /golem/work
+VOLUME /golem/work
+COPY Dockerfile /golem/info/description.txt
+COPY Dockerfile /golem/work/info.txt
+```
+
+Note we copy Dockerfile content into 2 different locations:
+
+- to /golem/info (this folder is not defined as VOLUME)
+- and to /golem/work (this folder is defied as VOLUME)
+
+## Building the Docker image
+
+To build the Docker image from the `Dockerfile`, we can run the following command in the same directory (`.`) as the Dockerfile to build an image tagged `golem-node`:
+
+{% tabs %}
+{% tab label="Linux" %}
+
+```bash
+docker build -t golem-node .
+```
+
+{% /tab %}
+{% tab label="macOS" %}
+
+```bash
+docker build --platform linux/amd64 -t golem-node .
+```
+
+{% /tab %}
+{% tab label="Windows" %}
+
+```bash
+docker build -t golem-node .
+```
+
+{% /tab %}
+{% /tabs %}
+
+The output should look like this:
+
+![](/image_tutorial_build.png)
+
+{% alert level="info" %}
+
+Note that the image won't be turned into a file in the same directory. The location of the actual Docker image file depends on the Docker and your operating system version. Please consult the Docker manual for additional information.
+
+{% /alert  %}
+
+## Converting from Docker to Golem and uploading it to the registry
+
+Now when you have a Docker image built, we can convert it to a Golem image. To save time, we will also upload it to the registry with the same command. To do this, you need to run the appropriate command that uses `gvmkit-build` to convert and push the image `golem-node` to the registry.
+
+{% alert level="info" %}
+
+If you do not have `gvmkit-build` installed please follow [installation intructions](/docs/creators/javascript/examples/tools/gvmkit-build-installation). You can also use it without installation using `npx` or `pipx` commands.
+
+{% /alert  %}
+
+{% tabs %}
+{% tab label="JavaScript" %}
+
+```bash
+gvmkit-build golem-node --push --nologin
+```
+
+{% /tab %}
+{% tab label="Python" %}
+
+```bash
+gvmkit-build golem-node --push --nologin
+```
+
+{% /tab %}
+{% /tabs %}
```

Comment:
these tabs are useless here as the commands are the same... 

```suggestion
```bash
gvmkit-build golem-node --push --nologin
```
```